src/main/java/core/basesyntax/FigureDrawer.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureDrawer {
```

Comment:
```suggestion
public interface Drawer {
```

мы можем использовать этот интерфейс не только для фигур

Reply:
I did it.